benchmarks/copy_view_performance/Makefile

```diff
@@ -0,0 +1,46 @@
+KOKKOS_DEVICES=Serial
+KOKKOS_ARCH = ""
+
+
+MAKEFILE_PATH := $(subst Makefile,,$(abspath $(lastword $(MAKEFILE_LIST))))
+
+ifndef KOKKOS_PATH
+  KOKKOS_PATH = $(MAKEFILE_PATH)../..
+endif
+
+SRC = $(wildcard $(MAKEFILE_PATH)*.cpp)
+HEADERS = $(wildcard $(MAKEFILE_PATH)*.hpp)
+
+vpath %.cpp $(sort $(dir $(SRC)))
+
+default: build
+	echo "Start Build"
+
+CXX = clang++
+EXE = copy_view_performance.exe
+
+CXXFLAGS ?= -Ofast -DKOKKOS_IMPL_REF_COUNT_BRANCH_UNLIKELY
```

Comment:
This macro should now be "KOKKOS_ENABLE_IMPL_REF_COUNT_BRANCH_UNLIKELY" to be the same as the macro in core/src/impl/Kokkos_SharedAlloc.hpp